tests/conftest.py

```diff
@@ -485,3 +485,14 @@ def mining_schema_df():
     schema_file = ROOT_PATH / "tests" / "data" / "mining" / "request" / "request.csv"
     df = pd.read_csv(schema_file)
     return df
+
+
+@pytest.fixture()
+def pubmed_articles_zipped_path(test_data_path, tmp_path):
```

Comment:
Up to you, but maybe one could use a shorter name? Seeing it causes line breaks down the line. Somethine like `pubmed_xml_gz` or similar?

Reply:
Done in [581588c](https://github.com/BlueBrain/Search/pull/582/commits/581588c91e530182c1d8f9c4a2e2dfac3ba5bd7b)